src/fluence/fluxion/fluxion.go

```diff
@@ -20,11 +20,11 @@ type Fluxion struct {
 }
 
 // InitFluxion creates a new client to interaction with the fluxion API (via go bindings)
-func (fluxion *Fluxion) InitFluxion(policy *string, label *string) {
+func (fluxion *Fluxion) InitFluxion(policy string, label string) {
```

Comment:
It does seem more opinionated than anything else ([another reddit post](https://www.reddit.com/r/golang/comments/113950u/pointer_or_not/)). I tend to follow what I see done most often in the ecosystem (e.g., in [Kubernetes](https://github.com/kubernetes/kubernetes/blob/bb95d084a23469614cb64a4b3d408027ac82f317/staging/src/k8s.io/kms/internal/plugins/_mock/plugin.go#L39) they tend to pass a regular string after `flag.Parse()`) but I treat it more as a message to a developer if something is intended to change. Specifically for pointers (on structs or other) this piece of wisdom from the reddit post:

> In a similar vein, don't default to pointer receivers unless you really mean to tell the caller that calling that function might change the value it is being called on. Pointer receivers are only necessary to avoid copying a sync.Mutex, to mutate the underlying value, or to avoid copying when its proven copying is actually a problem

It's (to me) a message that the thing is intended to be mutated and used again. Anyway, I'm OK to change it back if that is your preference. It's not mine but I don't feel strongly about it - let me know what you'd like to do!

Reply:
For the record: I'm fine with the code the way it is currently.